docs/anchor-platform.md

```diff
@@ -1,4 +1,53 @@
 # Anchor Platform Spring Boot Application
 
 ## To Run
-./gradlew bootRun
+```shell
+./gradlew bootRun -PjvmArgs="-Dstellar.anchor.config=path-to-yaml" 
```

Comment:
Is this path absolute or relative?

Reply:
Can be either absolute or relative depends on the file system. 

eg: `//home/user/.anchor/anchor-config.yaml` is absolute
eg: `./anchor/anchor-config.yaml` is relative.